app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+
+class AuthController extends Controller
+{
+    public function login(Request $request){
+
+        $creds = $request->validate([
+            'email' => 'required|email',
+            'password' => 'required'
+        ]);
+
+
+        if (Auth::attempt($creds)){
+
+            $token = Auth::user()->createToken('myToken')->plainTextToken;
+
+            $response = [
+                'user' => Auth::user(),
+                'token' => $token
+            ];
+
+            return response($response, 200);
```

Comment:
use response()->json($response)

200 status code is default, you dont have to write it